app.py

```diff
@@ -21,10 +21,10 @@ def predict():
     #predict = prediction[:, 1]

```

Comment:
GPT summary of [1912d4](https://github.com/SampatSuthar/ml-demo/blob/626c9a4f9de8e0a8a924632766ba55eefeaa321d/app.py#1912d43f95c066b874d8646e52a77052c42facc9) - [1fcf8a](https://github.com/SampatSuthar/ml-demo/blob/fd0a039f7bdb837ef624332a2cca21f9452a6f95/app.py#1fcf8aed8558b2618a97636c2f6d6c3587c2a6ee):
* Imported OpenAI library
* Updated OpenAI API key
* Changed model from "davinci-003" to "text-davinci-003"
* Created OpenAI completion with updated model and prompt
* Stored generated text from response in variable